pkg/ingester/user_state.go

```diff
@@ -362,9 +362,11 @@ type metricCounterShard struct {
 type metricCounter struct {
 	limiter *Limiter
 	shards  []metricCounterShard
+
+	ignoredMetricsForSeriesCount map[string]struct{}
```

Comment:
[nit] I think we can shorten the name. This struct only count series by metric name.
```suggestion
	ignoredMetrics map[string]struct{}
```